core/src/main/scala/org/apache/spark/util/collection/OpenHashSet.scala

```diff
@@ -85,9 +85,13 @@ class OpenHashSet[@specialized(Long, Int) T: ClassTag](
   protected var _capacity = nextPowerOf2(initialCapacity)
   protected var _mask = _capacity - 1
   protected var _size = 0
+  protected var _occupied = 0
   protected var _growThreshold = (loadFactor * _capacity).toInt
+  def g: Int = _growThreshold
+  def o: Int = _occupied
```

Comment:
Also, please use more descriptive and comprehensible names

Reply:
Oh, sorry for putting this. This is used only for my debugging. This should be removed.